src/main/java/org/miracum/streams/fhirtoserver/SendToServerProcessor.java

```diff
@@ -99,6 +100,14 @@ public SendToServerProcessor(
                 context.getRetryCount(),
                 throwable);
             sendingFailedCounter.increment();
+            if (throwable instanceof BaseServerResponseException fhirException) {
```

Comment:
So besser?

Reply:
Fehlt glaube noch ein Log von der Ausgabe in Zeile 105. *Falls* der Json blob der da zurückkommt schwer zu lesen ist in logausgaben kann man sonst die operationoutcomd.issues durchiterieren und einen loggen.

Reply:
🙈 selbstverständlich fehlt der log.
Ich bin mir nicht so sicher, wie ich den am besten testen kann, sonst wäre es mir wahrscheinlich selbst aufgefallen.

Reply:
Gute Frage. Ggf in unit test https://github.com/miracum/kafka-fhir-to-server/blob/master/src/main/java/org/miracum/streams/fhirtoserver/SendToServerProcessor.java#L65 mit gemocktem igenericclient der beim Ausführen der Transaktion eine exception wirft. Dann kann man zumindest manuell schnell sehen ob was geloggt wird - man kann theoretisch auch stdout capturen und asserten dass das was richtiges drinsteht... Lohnt sich aber vmtl nicht der Aufwand.